### ISSUE TITLE
The stream displays the first frame and freezes

### ISSUE THREAD

**Issue by PanCrucian** [ant-media/LibRtmp-Client-for-Android#51]
I use your library together with ExoPlayer v2.5.4, and when I try to play this stream _**rtmp://188.130.252.20:1935/live/test live=1**_ the stream displays the first frame and freezes

logcat:
```
I/MediaCodec: [OMX.qcom.video.decoder.avc] setting surface generation to 18942978
I/ExtendedACodec: setupVideoDecoder()
I/ExtendedACodec: vpp-enable search is 0 and value is 0
I/ExtendedACodec: Decoder will be in frame by frame mode
W/ExtendedACodec: Failed to set frame packing format on component
W/ExtendedACodec: Failed to get Profile and Level from Component
W/ExtendedACodec: Failed to get Entropy from Component
```

**Comment by PanCrucian** [comment]
Please try it (stream link) yourself

**Comment by PanCrucian** [comment]
It seems that the media server does not respond some meta data which should be in the RTMP stream. Can u @mekya please check and approve this?

**Comment by mekya** [comment]
Yeah I have checked and you are right. It seems that it is a codec issue in ExoPlayer.

**Comment by PanCrucian** [comment]
@mekya can you please help me find the issue i'm not well on this technology? or can tell me where to look in ExoPlayer

**Comment by mekya** [comment]
I think we may request help from one of the developer of Exoplayer  

Is it possible to take a look at this issue @ojw28?

If ojw28 is busy to take a look at this issue, I can dig into this and try to help you.

**Comment by PanCrucian** [comment]
yeah @ojw28 can u please? I opened [new ticket](https://github.com/google/ExoPlayer/issues/3402) in [ExoPlayer project](https://github.com/google/ExoPlayer)

**Comment by amankumarjain** [comment]
@PanCrucian hi can you please share your code, how you have implemented rtmp live stream video suing this, i can't find doc for this
